CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+
+A good bug report is one that makes it easy for everyone to understand what you were trying to do and what went wrong. Provide as much context as possible so we can try to recreate the issue.
+
+## Contributions 
+
+Contributing to an Open Liberty guide is simple. All you need to do is create your own branch to include your contribution and open a pull request to the `qa` (quality assurance) branch.
+
+### Text or Content Contributions
+
+If you are contributing text of content changes via a pull request and you certify the [Developer Certificate of Origin (DCO)](https://developercertificate.org), add a line to the end of the Git commit message to sign your work:
+
+```
+Signed-off-by: Jane Williams <jane.williams@gmail.com>
+```
+
+The sign-off is just a line at the end of the commit message that certifies that you wrote it or otherwise have the right to pass it on as an open source patch.
+
+Use your real name when you sign. We can't accept pseudonyms or anonymous contributions. 
+
+Many Git UI tools have support for adding the `Signed-off-by` line to the end of your commit message. This line can be automatically added by the `git commit` command by using the `-s` option.
+
+If reviewers agree with your change, your change will be merged to the `master` branch for publishing.
+
+### Code Contributions
+
+If you are contributing trivial changes such as typos, redundant spaces, minor formatting and spelling errors, you must follow the above instructions under [Text or Content Contributions](./CONTRIBUTING.md#text-or-content-contributions).
+
+If you are contributing code changes via a pull request for anything except trivial changes, you must signoff on the [Individual Contributor License Agreement](https://github.com/OpenLiberty/open-liberty/blob/master/cla/open-liberty-cla-individual.pdf) If you are doing this as part of your job you may also wish to get your employer to sign a CCLA [Corporate Contributor License Agreement](https://github.com/OpenLiberty/open-liberty/blob/master/cla/open-liberty-cla-corporate.pdf). Instructions how to sign and submit these agreements are located at the top of each document. 
+
+After we obtain the signed CLA, you are welcome to open a pull request against the `qa` branch, and the team will be notified for review. We ask you follow these steps through the submission process.
+
+1. Open a pull request against the `qa` branch.
+2. A "CLA signed" label will be added.
+3. The team will be notified and will review your change. 
+    - If there are further changes to be made, the team will request changes on the pull request. 
+    - If the team does not agree with the change, the PR will be closed with an explanation and suggestion for follow-up.
+4. If the team approves, a full Open Liberty site build will be invoked.
+5. Based on the results of the build, and if further review is needed, more discussion will occur.
```

Comment:
"more discussion will occur" -> "we will discuss it with you" or something like that - be more specific about who will do what so it's clearer what's happening if things go quiet for a bit.

Reply:
Addressed:
```
5. Based on the results of the build:
    - If further review is needed, we will let you know about a pending review from our team and discuss any necessary improvements that need to be made to your change(s). 
    - If everything is successful, the team will merge your PR.
```